server/test/assetComputeDevTool.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+'use strict';
+const assert = require('assert');
+// const { promisify } = require('util');
+// var exec = promisify(require('child_process').exec);
+const mockRequire = require('mock-require');
+
+describe( 'assetComputeDevTool.js tests', () => {
+	afterEach(() => {
+		mockRequire.stopAll();
```

Comment:
I saw you correct it in other files - Indentation here also seems to be too large (feels like8 instead of 4 spaces).